### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by karenc-bq**
### Summary

Update changelog for the upcoming release

### Additional Reviewers

@joyc-bq 
@aaronchung-bitquill 
@brunos-bq

### By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
 
+## [1.1.0] - 2023-01-17
+### Added
+- [Read / Write Splitting and Load Balancing (Preview)](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/7a7b8aa40f80173b19e2af8de6eedb2f93fd3b3d/docs/using-the-jdbc-driver/using-plugins/UsingTheReadWriteSplittingPlugin.md). On-going changes are being implemented to mirror behaviour of the community MySQL JDBC driver. We do not recommend using this in production until an official release.
+- The Aurora Stale DNS Plugin.
```
Comment (brunos-bq) [inline CHANGELOG.md:10]: should it have some words, like...

Aurora Stale DNS Plugin for "this" and "that"

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
...
+- FailoverSQLException classes for easier error handling. See example [here](./examples/AWSDriverExample/src/main/java/software/amazon/ReadWriteSplittingPostgresExample.java).
+- OSGi compatibility.
```
Comment (brunos-bq) [inline CHANGELOG.md:12]: any OSGi version or a specific one?

**Review by sergiyvamz — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (karenc-bq) [reply CHANGELOG.md:12]: any

File: CHANGELOG.md (hunk 1 above)
Comment (karenc-bq) [reply CHANGELOG.md:10]: > I think we are removing this, no ?

@davecramer We added the logic to the failover plugin but we are still keeping this plugin.

**Pull request merged**